templates/dashboard/staff/detail.html

```diff
@@ -70,7 +68,6 @@
                 </div>
```

Comment:
Shouldn't it better to call the template `form` instead of `detail`? It's kind of confusing when looking for the form template as the template is mostly a form.